TWLight/users/models.py

```diff
@@ -158,109 +142,82 @@ class Meta:
     user = models.OneToOneField(settings.AUTH_USER_MODEL)
     # Set as non-editable.
     date_created = models.DateField(
-        default=now,
-        editable=False,
-        # Translators: The date the user's profile was created on the website (not on Wikipedia).
-        help_text=_("When this profile was first created"),
+        default=now, editable=False, help_text="When this profile was first created"
     )
 
     # ~~~~~~~~~~~~~~~~~~~~~~~ Data from Wikimedia OAuth ~~~~~~~~~~~~~~~~~~~~~~~#
     # Uses same field names as OAuth, but with wp_ prefixed.
     # Data are current *as of the time of last TWLight login* but may get out of
     # sync thereafter.
-    wp_username = models.CharField(max_length=235, help_text=_("Username"))
-    # Translators: The total number of edits this user has made to all Wikipedia projects
+    wp_username = models.CharField(max_length=235, help_text="Username")
     wp_editcount = models.IntegerField(
-        help_text=_("Wikipedia edit count"), blank=True, null=True
+        help_text="Wikipedia edit count", blank=True, null=True
```

Comment:
This is on user profile, I think.

Reply:
Ignore me, we duplicate this: https://github.com/WikipediaLibrary/TWLight/blob/6f6b88be087460fc32b9532448ee7b4bc458f37e/TWLight/users/templates/users/editor_detail_data.html#L114